CHANGES.rst

```diff
@@ -24,6 +24,8 @@ Improvements
   (:pr:`6860`)
 - Use :data:`STATIC_SITE_STORAGE` for the temporary file from a material package
   (:pr:`6898`)
+- Log user merges to user
+  (:issue:`6882`, :pr:`6920`)
```

Comment:
```suggestion
- Log user merges to the target's user log (:issue:`6882`, :pr:`6920`)
```

The wrapping in the lines above is just because we try to not make them too long, and for these changelog entries it's nice to be just slightly more verbose.

FWIW, this kind of change is small enough to not really *require* a changelog entry, but nothing wrong with having one. :)

Reply:
I think the suggested change to the message itself somehow got lost